install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -233,13 +241,59 @@ spec:
         {{- end }}
         {{- end }}
         securityContext:
+          {{- if .Values.securityContext.privileged }}
           privileged: true
+          {{- else }}
+          seLinuxOptions:
+            level: 's0'
+            # Running with spc_t since we have removed the privileged mode.
+            # Users can change it to a different type as long as they have the
+            # type available on the system.
+            type: 'spc_t'
```

Comment:
Does this mean that privileged=false requires SELinux?

Reply:
No no, it only means that if SELinux is available / enabled then we should be running the container with level `s0` and type `spc_t`